### PR TITLE
Update Maude and Libbuddy Bottles for Big Sur

### PR DESCRIPTION
**Pull request by mpdehnel**
I've built Maude 2.7.1 and Libbuddy 2.4 for macOS Big Sur. I think the the formulae are now correct compared to the updated bottles (submitted to https://github.com/tamarin-prover/binaries/) — please shout if not!
Please see https://github.com/tamarin-prover/binaries/pull/1 for the paired PR for the bottles.

### CODE REVIEW

**Comment by mpdehnel** [comment]
Ok, I've tested this locally (as `brew install mpdehnel/tap/tamarin-prover`) and this works perfectly, although it only uses the catalina/mojave bottles for libbuddy and maude respectively, as it was otherwise conflicting with the `tamarin-prover/tap` copy vs the `mpdehnel/tap` versions:
```
==> Installing tamarin-prover from mpdehnel/tap
Error: Formulae found in multiple taps: 
       * tamarin-prover/tap/libbuddy
       * mpdehnel/tap/libbuddy

Please use the fully-qualified name (e.g. tamarin-prover/tap/libbuddy) to refer to the formula.
```
...so I could only use the existing `maude.rb` and `libbuddy.rb` files in `tamarin-prover/tap/` rather than my updated versions with the correct hashes for `big_sur`.

**However,** I think when merged, this should all work fine... if you can see a better way to test it then please let me know! Otherwise we can try merging and roll back / squash it if it doesn't work.

Feel free to squash the commits! Many were just for my local testing purposes.

File: Formula/libbuddy.rb [hunk 1]
```diff
@@ -7,11 +7,8 @@ class Libbuddy < Formula
   bottle do
     root_url "https://raw.githubusercontent.com/tamarin-prover/binaries/HEAD/dependencies"
     rebuild 3
+    sha256 cellar: :any_skip_relocation, big_sur:      "2d2c92c55832c9f6ba160cc9154542779a63ab111bb8bc668f295e776dc1ab9a"
     sha256 cellar: :any_skip_relocation, catalina:     "3cab96ab2fe4506669abd447bf5185e789ab0ea2a40536ff61b0b734f167f5a0"
-    sha256 cellar: :any_skip_relocation, high_sierra:  "5c150e653aeb36ce34381f24137c963419a169e665cdfa5e6f15495923beb694"
-    sha256 cellar: :any_skip_relocation, el_capitan:   "5dc396c196d46646102d10c5d3ff32d8b37249cac78af2595417e6474494453b"
-    sha256 cellar: :any_skip_relocation, yosemite:     "451c34f54e575578a3b3cfaad5e5a860f012317ff32053097c6132bba60a7da9"
-    sha256 cellar: :any_skip_relocation, mavericks:    "f4a66068fc8b53b557e3c9f6be57574c1711aedd91e1f18e2b6c5111b62fdbe7"
```
Comment (mpdehnel) [inline Formula/libbuddy.rb:14]: These bottles no longer exist in https://github.com/tamarin-prover/binaries/tree/main/dependencies hence removing them

**Comment by jdreier** [comment]
Looks good, thank you!

**Comment by mpdehnel** [comment]
With the update to binaries (file name) this now works perfectly on a fresh Big Sur install.

**Pull request merged**